content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -75,6 +75,12 @@ For Linux distributions other than RHEL, check Docker compatibility:
 
 ~> **Note**: It is not recommended to run Docker under a 2.x kernel.
 
+#### SELinux
+
+At present the product does not support SELinux. The host running the installer must be configured in permissive mode by running: `setenforce 0`.
```

Comment:
I think I'd rather we say this as "Private Terraform Enterprise does not..." (applies to the migration page as well)
I'm not wild about the "future releases" phrasing below either, but since it is a common ask it seems acceptable here.